backends/gstreamer/lib.rs

```diff
@@ -49,12 +51,23 @@ use servo_media_streams::capture::MediaTrackConstraintSet;
 use servo_media_streams::registry::MediaStreamId;
 use servo_media_streams::MediaOutput;
 use servo_media_webrtc::{WebRtcBackend, WebRtcController, WebRtcSignaller};
+use std::collections::HashSet;
 
 lazy_static! {
     static ref BACKEND_BASE_TIME: gst::ClockTime = { gst::SystemClock::obtain().get_time() };
 }
 
-pub struct GStreamerBackend;
+pub struct GStreamerBackend {
+    registry_scanner: GStreamerRegistryScanner,
```

Comment:
Let's add some docs about what the `registry_scanner` is. i.e.: `The GStreamer registry holds the metadata of the set of plugins available in the host. This scanner is used to lazily analyze the registry and to provide information about the set of supported mime types and codecs that the backend is able to deal with`